stellar_multisig-rails.gemspec

```
@@ -9,14 +9,18 @@ Gem::Specification.new do |s|
   s.version     = StellarMultisig::Rails::VERSION
   s.authors     = ["Ramon Tayag"]
   s.email       = ["ramon.tayag@gmail.com"]
-  s.homepage    = "TODO"
-  s.summary     = "TODO: Summary of StellarMultisig::Rails."
-  s.description = "TODO: Description of StellarMultisig::Rails."
+  s.homepage    = "https://github.com/bloom-solutions/stellar_multisig-rails"
+  s.summary     = "Use Stellar Multisignatures to easily secure wallets"
+  s.description = "Description of StellarMultisig::Rails."
   s.license     = "MIT"
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", "~> 5.2.1"
+  s.add_dependency "rotp"
+  s.add_dependency "addressable", "~> 2.5.2"
```

We should make this less strict so that if this is installed in an existing Rails app, it'll be less likely to fight with other versions:

```suggestion
  s.add_dependency "addressable"
```

We can just add versions when we know a some versions will not work with out code base.